src/components/02-components/card-expreciences/card-experiences.tsx

```diff
@@ -0,0 +1,230 @@
+import { Button, Box, Paper, Typography } from '@mui/material';
+import { useTheme } from '@mui/material/styles';
+import CardNumber from '../card-number/card-number';
+
+export type BrandProps = {
+  heading?: string;
+  state: string;
+  description?: string;
+  program?: string;
+  dateStart: string;
+  dateEnd: string;
+  location?: string;
+  cta: string;
+  ctaTitle: string;
+  hours: string;
+  hoursText: string;
+  hoursCtaTitle: string;
+  hoursCtaUrl: string;
+  variation: number;
```

Comment:
`variation` is the card count right? In that case, is better to change the name to something more explicit (like `cardCount`).